app/addons/documents/routes-index-editor.js

```diff
@@ -28,40 +28,53 @@ define([
   'addons/documents/pagination/pagination.react',
   'addons/documents/header/header.react',
   'addons/documents/header/header.actions',
+  'addons/documents/sidebar/actions'
 ],
 
 function (app, FauxtonAPI, Helpers, BaseRoute, Documents, IndexEditorComponents, ActionsIndexEditor,
           Databases, Components, IndexResultsStores, IndexResultsActions,
-          IndexResultsComponents, ReactPagination, ReactHeader, ReactHeaderActions) {
+          IndexResultsComponents, ReactPagination, ReactHeader, ReactHeaderActions, SidebarActions) {
 
 
   var IndexEditorAndResults = BaseRoute.extend({
-    layout: 'two_pane',
+    layout: 'with_tabs_sidebar',
     routes: {
-      'database/:database/new_view': 'newViewEditor',
-      'database/:database/new_view/:designDoc': 'newViewEditor',
+      'database/:database/new_view': {
+        route: 'createView',
+        roles: ['fx_loggedIn']
+      },
+      'database/:database/new_view/:designDoc': {
+        route: 'createView',
+        roles: ['fx_loggedIn']
+      },
       'database/:database/_design/:ddoc/_view/:view': {
-        route: 'viewFn',
+        route: 'showView',
+        roles: ['fx_loggedIn']
+      },
+      'database/:database/_design/:ddoc/_view/:view/edit': {
```

Comment:
hmm.. changing the route diverges us more from the couchdb routing:

right now

```
http://localhost:8000/#/database/blerg/_design/yxc/_view/new-view


           http://localhost:5984/blerg/_design/yxc/_view/new-view
```

that makes it easy to copy / paste stuff around for debugging / development of applications that use couchdb views, i can click to the view in the fauxton ui and then copy the second part of the url 1:1 into my application code